External/CoordGen/CMakeLists.txt

```diff
@@ -85,19 +98,15 @@ if(RDK_BUILD_COORDGEN_SUPPORT)
     set(coordgen_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}
         CACHE STRING "CoordGen Include File" FORCE)
     file(GLOB CGSOURCES "${COORDGEN_DIR}/*.cpp")
-    rdkit_library(coordgen ${CGSOURCES} SHARED LINK_LIBRARIES ${maeparser_LIBRARIES})
+    rdkit_library(coordgen ${CGSOURCES} SHARED )
     install(TARGETS coordgen DESTINATION ${RDKit_LibDir})
     set(coordgen_LIBRARIES coordgen)
 
   endif(COORDGEN_FORCE_BUILD OR (NOT coordgen_FOUND))
 
-  include_directories(${maeparser_INCLUDE_DIRS})
   include_directories(${coordgen_INCLUDE_DIRS})
 
-  install(FILES DESTINATION ${RDKit_ShareDir}/Data
-          COMPONENT data)
-
-  set(RDK_COORDGEN_LIBS MolAlign ${coordgen_LIBRARIES} ${maeparser_LIBRARIES} ${Boost_LIBRARIES}
+  set(RDK_COORDGEN_LIBS MolAlign ${coordgen_LIBRARIES} ${RDK_MAEPARSER_LIBS} ${Boost_LIBRARIES}
```

Comment:
I think we should also drop all this templatefiledir stuff here: https://github.com/rdkit/rdkit/blob/master/External/CoordGen/CoordGen.h#L52

But that may make sense as a separate commit.